src/app/relay/mutations/UpdateProjectMediaMutation.js

```diff
@@ -38,14 +37,25 @@ class UpdateProjectMediaMutation extends Relay.Mutation {
           demand
           requests_count
           linked_items_count
-          project_id,
           overridden,
           metadata,
           dbid,
           log,
           log_count,
           archived,
           permissions,
+          projects(first: 10000) {
+            edges {
+              node {
+                id
+                dbid
+                title
+                search_id
+                search { id, number_of_results }
+                medias_count
+              }
+            }
+          },
```

Comment:
Do you really need to re-fetch the whole list of projects as a result of the mutation?

Reply:
no I removed this list